packages/keyboard-keys/src/keys.ts

```diff
@@ -13,6 +13,7 @@ export const SymbolLock = 'SymbolLock';
 export const Hyper = 'Hyper';
 export const Super = 'Super';
 export const Enter = 'Enter';
+export const Space = ' ';
```

Comment:
not fully understand what exactly happening here, but believe it is intentional change ? :)

Reply:
yeah `Space` was missing